rllib/agents/ppo/ddppo.py

```diff
@@ -186,6 +202,126 @@ def validate_config(self, config):
         if config["kl_coeff"] != 0.0 or config["kl_target"] != 0.0:
             raise ValueError("DDPPO doesn't support KL penalties like PPO-1")
 
+    @override(PPOTrainer)
+    def setup(self, config: PartialTrainerConfigDict):
+        super().setup(config)
+
+        # Initialize torch process group for
+        if self.config["_disable_execution_plan_api"] is True:
+            self._curr_learner_info = {}
+            ip = ray.get(self.workers.remote_workers()[0].get_node_ip.remote())
```

Comment:
Could we also use local worker here to have one single point of failure and not two (driver process and remote_worker #0?
And then 270-280 could also go?
Bein curious here, not an expert.